packages/core/src/scp-cf/destination-accessor.ts

```diff
@@ -207,25 +216,37 @@ export async function getDestinationFromDestinationService(
 }
 
 function tryDestinationForServiceBinding(name: string): Destination | undefined {
+  logger.info('Attempting to retrieve destination from service binding.');
   try {
-    return destinationForServiceBinding(name);
+    const destination = destinationForServiceBinding(name);
+    logger.info('Sucessfully retrieved destination from service binding.');
+    return destination;
   } catch (error) {
-    logger.warn(error.message);
-    logger.warn("If you're not using SAP Extension Factory, you can ignore this warning.");
+    logger.info(error.message);
```

Comment:
What about the stacktrace? Could be useful.

Reply:
I changed this from error to info, which will look less scary (people often confuse it with an actual error). In my opinion there should ne error be thrown at all, this is just an information that a destination could not be found from the service binding (which will be the case in most cases). Therefore I don't think a stackstrace will be very helpful, what do you think?

Reply:
ah, ok I understand - no then it is fine. This is just step 2. of
1. check env
2. check serviceBinding
3. check destination service

See my comment below regarding step 1.

Reply:
correct.

Reply:
I guess the problem is that the user has to interpret whether it's a warning or not, so I'm fine either way. Also I'm afraid some users will still be confused, regardless of the channel.